### ISSUE TITLE
Zoom resets when "autoreload" is selected.

### ISSUE THREAD

**Issue by MalakymR** [bfabiszewski/ulogger-server#230]
When autoreload is selected, the page resets the zoom level each time, which is especially difficuilt when reload is fast (1 second etc).

This zoom level is also too close for ESRI to show images, and too close in general if a default needs to be selected.

Using 1.2 which suggsted this was fixed with https://github.com/bfabiszewski/ulogger-server/commit/f641b68e144cbed9fcd509a5fe8e2f6b9d05f587 but does not seem to be the case.

**Comment by bfabiszewski** [comment]
That is strange, because it works fine for me. 
Maybe this is browser specific? Please check in different browsers.
Does it happen in both OpenLayers and GoogleMaps API?

Regarding zoom level, the application does not know which is minimum level for given layer. I will have to change the way the layers are added by user, so that user will also have to provide minimum level for added layer.

**Comment by MalakymR** [comment]
Firefox 126.0.1 (inc Mobile, 126.x), Edge 125.x

The auto zoom happens with GoogleMaps API too (though zooms out to a more usable viewpoint).

The webui does show 1.2 in the corner, I also checked the code to make sure it had updated and matches the changes.

**Comment by bfabiszewski** [comment]
Do you have "latest position" turned on? I now realized it still does not work when this option is on.

**Comment by MalakymR** [comment]
Yes I do - I had mistakenly assumed this is what the last update fixed.

The zoom change without latest position only seems to occur with a new position, sometimes...

**Comment by bfabiszewski** [comment]
I don't use latest position, so I forgot about it.
I am now in the middle of rewriting the whole app, so I will fix it in the next version.

Without latest position it works fine for me. Zoom never changes, only view is moved when new position is put outside the window.

**Comment by bfabiszewski** [comment]
BTW, instead of ESRI layer (which I included as an example), you can add other satellite tiles, like Mapbox, which has lower minimum zoom:
https://api.mapbox.com/v4/mapbox.satellite/{z}/{x}/{y}@2x.jpg90?access_token=xxxxxxx
You just need to register to get free token.
